notebooks/01-variables.ipynb

```diff
@@ -17,12 +17,12 @@
   },
```

Comment:
<p>Aqui podemos poner *equivocarse en altura_medicion. Escribir Altura_medicion para que vean el error. Seguramente me equivocare sin querer pero si no tenemos esta nota. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoLatinas/Intro-to-python/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>El tema de _case sensitive_ aparece más abajo y además dejé una discusión que esconde ese error como para que lo mostremos y no nos olvidemos jeje.</p><p> </p><p>Si nos llegamos a equivocar antes lo podemos mostrar ahí mismo, pero quizás para el notebook de guía me gustaba dejarlo más al final y acá hacer incapié en cómo podemos guardar resultados en nuevas variables. ¿Qué te parece?</p><p> </p>

Reply:
oki! de acuerdo. Ya lo vi donde aparece. Gracias